.github/workflows/ci.yml

```diff
@@ -188,7 +188,7 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: 7.3
-                  extensions: dom, fileinfo, filter, gd, hash, intl, json, mbstring, pcre, pdo_mysql, zlib
+                  extensions: dom, fileinfo, filter, gd, hash, intl, json, mbstring, mysqli, pcre, pdo_mysql, zlib
```

Comment:
Why is this necessary?

Reply:
The test fails on Windows without it. As I said, I am not sure why the test did not fail in #2462 without it too.

Reply:
I see.